### DISCUSSION TITLE
IDE Query

### DISCUSSION THREAD

**Discussion by mbowler-oyova** [harness-software/wp-graphql-gravity-forms#251] [Q&A] [answered]
When trying to query the example from https://developers.wpengine.com/blog/work-with-gravity-forms-in-headless-wordpress, IDE gives me a cannot query gravityFormsForm on type "Root Query". Any suggestions? ID is correct at 5 and Questionnaire is active in forms.

**Comment by justlevine** [comment, marked as answer]
Hey @mbowler-oyova ,
While the _concepts_ in that article are good, it is based on an older (and no longer compatible) plugin version. Take a look [here](https://github.com/harness-software/wp-graphql-gravity-forms/blob/develop/docs/querying-forms.md) and in the other docs for up-to-date example queries.)
(E.g. `gravityFormsForm` is now `gfForm`

**Reply by mbowler-oyova** [reply]
That helps. Thank you.
